.changeset/silly-owls-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": minor
+---
+
+Version Update for "Manuale dei Servizi" from 2.3 to 2.4
```

Comment:
You should always describe changes (like commit messages) in the imperative mood, like you're giving an order. For example, “Update version” instead of “Updated version”, “Updating version" or "Version Update" (which is not an action).
In this particular case you are adding a new version so the change description could be `Add version 2.4 of "Manuale dei Servizi"`  or `Update version for "Manuale dei Servizi" from 2.3 to 2.4`